products/eclipse-temurin.md

```diff
@@ -32,20 +32,20 @@ auto:
 # EOL dates can be found on https://adoptium.net/support/.
 releases:
 -   releaseCycle: "19"
-    releaseDate: 2021-12-13
+    releaseDate: 2022-10-26
```

Comment:
Looks like `latest.py` skipped `"19+36": "2022-09-26"` and used `"19.0.1+10": "2022-10-26"` for the release date in https://github.com/endoflife-date/release-data/blob/main/releases/eclipse-temurin.json (looks the same for all releases).